float32[4] joint
string gripper
float32 time